capsicum-test.cc

```diff
@@ -50,7 +50,11 @@ char ProcessState(int pid) {
 #endif
 #ifdef __FreeBSD__
   char buffer[1024];
-  snprintf(buffer, sizeof(buffer), "ps -p %d -o state | grep -v STAT", pid);
+  /* 
+   * XXX: -a is used with ps(1) as a temporary workaround for retrieving the
```

Comment:
Could you make the `XXX` into `TODO(sivamahadevan)` or `TODO(#123)` (where 123 is the number of an issue you raise to track this) please?